File: FizzBuzz.cpp
#include <iostream>
using namespace std;

int main()
{
	for (unsigned char c = 1; c < 101; ++c)
	{
		if (c % 15 == 0) cout << "FizzBuzz" << endl;
		else if (c % 3 == 0) cout << "Fizz" << endl;
		else if (c % 5 == 0) cout << "Buzz" << endl;
		else cout << +c << endl;
	}
	return 0;
}

File: FizzBuzz.java
package fizzbuzz;

public class FizzBuzz
{
    public static void main(String[] args)
    {
        for (byte b = 1; b < 101; ++b)
        {
            if (b % 15 == 0) System.out.println("FizzBuzz");
            else if (b % 3 == 0) System.out.println("Fizz");
            else if (b % 5 == 0) System.out.println("Buzz");
            else System.out.println(b);
        }
    }
}
